libs/styles/src/lib/scss/generic/_animations.scss

```diff
@@ -0,0 +1,3 @@
+@keyframes loading-spinner {
```

Comment:
I think it would make sense to rename the animation to `cvi-loading-spinner`. All the animations are in the global scope, so if a consuming app adds its own animations, the prefix should allow to understand where a library animation comes from.